src/components/FAQ/QuestionForm.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div class="container">
+    <div>
+      <h5 class="text-center">Ask a Question</h5>
+      <form>
+        <div v-if="form_data.question.title.length">
+          <h6>Questions that may answer your query:</h6>
+          <div v-for="suggestion in suggestions">
+            <router-link :to="{name: 'faq_detail', params: { id: suggestion.id }}">{{suggestion.title}}</router-link>
+            <br/>
+          </div>
+          <br/>
+        </div>
+        <BaseInput placeholder="Question Title" v-model="form_data.question.title"
+                   :valid="form_validity.title"></BaseInput>
+        <div :class="this.form_validity.body == false ? 'has-danger': ''">
+                    <textarea :class="this.form_validity.body == false ? 'form-control is-invalid': 'form-control'"
```

Comment:
You don't have to use this. in the template
You could try https://github.com/vuejs/eslint-plugin-vue to lint the code if you want